src/init/features/project.ts

```diff
@@ -120,14 +120,22 @@ export async function doSetup(setup: any, config: any, options: any): Promise<vo
   }
 
   let projectMetaData;
-  // If the user presented a project with `--project`, try to fetch that project.
   if (options.project) {
+    // If the user presented a project with `--project`, try to fetch that project.
     logger.debug(`Using project from CLI flag: ${options.project}`);
     projectMetaData = await getFirebaseProject(options.project);
   } else {
-    projectMetaData = await projectChoicePrompt(options);
-    if (!projectMetaData) {
-      return;
+    const projectEnvVar = utils.envOverride("GCLOUD_PROJECT", "");
```

Comment:
Defer to @joehan on this~

Is this a reasonable env name?

Reply:
Okay switch to `FIREBASE_PROJECT`. Later maybe we can make it a drop-in replacement of `--project`. Or not. The script can work either way